lib/pipeline.ts

```diff
@@ -286,6 +293,23 @@ export class Pipeline extends Construct {
     if (props.autoBuild) {
       this.autoBuild(props.autoBuildOptions);
     }
+
+    if (props.failureNotifySlack && props.failureNotifySlack.length > 0) {
+      props.failureNotifySlack.forEach(s => {
+        new starnotifs.CfnNotificationRule(this, `FailureSlackNotification-${s.slackChannelConfigurationName}`, {
+          name: `${this.pipeline.pipelineName}-failednotifications`,
+          detailType: 'BASIC',
+          resource: this.pipeline.pipelineArn,
+          targets: [
+            {
```

Comment:
Wouldn't it have made more sense to put the array here instead of `forEach`ing over it? 

Or is this a fake array?